skimage/future/graph/rag.py

```diff
@@ -263,6 +266,125 @@ def rag_mean_color(image, labels, connectivity=2, mode='distance',
            http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.11.5274
 
     """
+
+
+    def define_graph(graph,labels,image, extra_arguments=[],
```

Comment:
The scope of the function is is local to `rag_mean_color()`.   Are you suggesting to call it `define_color_based_similarity` as the name?